### PR TITLE
Ethan/suggested links

### PR DESCRIPTION
**Pull request by etown**
Adds bing web search service and adds suggested links to conversations if it's enabled.

### CODE REVIEW

File: owl/prompts/summarization.py [hunk 1]
```diff
@@ -1,5 +1,9 @@
 from ..core.config import Configuration
 
+def suggest_links_system_message(config: Configuration) -> str:
```
Comment (trzy) [inline owl/prompts/summarization.py:3]: Move this into own file under prompts/. It's not technically summarization. Maybe web_search.py or suggested_links.py?

**Review by trzy — CHANGES_REQUESTED** [review]

File: owl/models/schemas.py [hunk 2]
```diff
@@ -99,6 +106,37 @@ class CaptureSegment(CreatedAtMixin, table=True):
 
     conversation: Optional[Conversation] = Relationship(back_populates="capture_segment_file")
 
+#  Bing Search API Response Models
```
[the remark this answers is not in the document]
Comment (etown) [reply owl/models/schemas.py]: Done

File: owl/prompts/summarization.py (hunk 1 above)
Comment (etown) [reply owl/prompts/summarization.py]: Done

**Review by trzy — APPROVED** [review]
LGTM subject to Alembic stuff being merged. Going to take a look at that PR.

File: owl/services/web_search/bing_search_service.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import httpx
+from ...models.bing import BingSearchResponse
+
+class BingSearchService:
+    def __init__(self, config):
```
Comment (trzy) [inline owl/services/web_search/bing_search_service.py:5]: Can you add a type annotation here to clarify that config is actually BingSearchConfiguration not Configuration?

Comment (etown) [reply owl/services/web_search/bing_search_service.py]: Done

**Pull request merged**